Pal/src/slab.c

```diff
@@ -135,13 +135,13 @@ void * malloc_copy (const void * mem, size_t size)
     return nmem;
 }
 
-char * strdup (const char *str)
+char * strdup (const char *s)
 {
-    size_t size = strlen(str) + 1;
-    char *new = malloc(size);
+    size_t len = strlen(s) + 1;
```

Comment:
Hmm, why did you revert this change? This variable doesn't hold the length of the string but its size. The name `len` is very misleading IMO.